src/helpers/shared.ts

```diff
@@ -19,3 +19,15 @@ export const shouldSkip = (): { skip: boolean; reason: string } => {
 };
 
 export const wait = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+export const calculateWeight = (label: LabelItem | undefined): number => {
+  if (!label) return 0;
+  const matches = label.name.match(/\d+/);
+  const number = matches && matches.length > 0 ? parseInt(matches[0]) : 0;
+  if (label.name.split(":")[0] === "Priority") return number + 1;
+  if (label.name.includes("Hour")) return number * 0.125;
+  if (label.name.includes("Day")) return 1 + (number - 1) * 0.25;
+  if (label.name.includes("Week")) return number + 1;
+  if (label.name.includes("Month")) return 5 + (number - 1) * 8;
```

Comment:
Can you explain the math formulas?

Reply:
I made the formulas to calculate weight from time-label name.
1Hour: 0.125
1Day: 1
1Week: 2
https://github.com/ubiquity/ubiquibot/blob/59825e4d7289a8da87bb3f568d9e69b183c821cf/ubiquibot-config-default.json#L16C8-L16C8